### PR TITLE
Disable a site completely

### PR DESCRIPTION
**Pull request by RickDBCN**
Adds basic support for #1

### CODE REVIEW

File: database/migrations/2022_09_16_081901_create_sites_table.php [hunk 1]
```diff
@@ -21,6 +21,7 @@ public function up()
             $table->string('public')->nullable();
             $table->boolean('track_clicks')->default(true);
             $table->boolean('track_movements')->default(true);
+            $table->boolean('active')->default(true);
```
Comment (Cannonb4ll) [inline database/migrations/2022_09_16_081901_create_sites_table.php:24]: Please create a new migration to add this column, existing applications won't run this piece of code and miss out on this column, or force them to refresh their database and loose data.

**Review by Cannonb4ll — COMMENTED** [review]
Please check reviewed files.

File: app/Filament/Resources/SiteResource.php [hunk 2]
```diff
@@ -63,6 +63,7 @@ public static function table(Table $table): Table
                 Tables\Columns\TextColumn::make('clicks_count')->counts('clicks')->label('Clicks')->sortable(),
                 Tables\Columns\TextColumn::make('movements_count')->counts('movements')->label('Movements')->sortable(),
                 Tables\Columns\TextColumn::make('created_at')->label('Date')->sortable()->dateTime('Y-m-d H:i:s'),
+                Tables\Columns\ToggleColumn::make('active')->label('Active')->sortable();
```
Comment (stayallive) [inline app/Filament/Resources/SiteResource.php:66]:
```suggestion
                Tables\Columns\ToggleColumn::make('active')->label('Active')->sortable(),
```

File: database/migrations/2022_09_16_081901_create_sites_table.php (hunk 1 above)
Comment (RickDBCN) [reply database/migrations/2022_09_16_081901_create_sites_table.php]: resolved

File: app/Filament/Resources/SiteResource.php (hunk 2 above)
Comment (RickDBCN) [reply app/Filament/Resources/SiteResource.php]: Resolved. I was a little bit too quick haha

**Pull request merged**